nengo/learning_rules.py

```diff
@@ -7,12 +7,10 @@
 
 
 class ConnectionParam(NengoObjectParam):
```

Comment:
This is also going to get removed in #1310, but I guess that's obvious enough that it isn't worth mentioning?